apps/judicial-system/backend/src/app/formatters/requestPdf.ts

```diff
@@ -257,19 +256,18 @@ function constructInvestigationRequestPdf(
   setLineGap(doc, 24)
   addHugeHeading(doc, title, 'Times-Roman')
   setLineGap(doc, 8)
-  addMediumPlusHeading(
-    doc,
-    `${formatDate(
-      theCase.created,
-      'PPP',
-    )} - Mál nr: ${theCase.policeCaseNumbers.join(', ')}`,
-  )
+  addMediumPlusHeading(doc, formatDate(theCase.created, 'PPP') ?? '')
   setLineGap(doc, 40)
   addMediumPlusHeading(
     doc,
     `${formatMessage(m.baseInfo.court)} ${theCase.court?.name}`,
   )
   setLineGap(doc, 4)
+  addMediumText(doc, 'LÖKE málsnúmer', 'Times-Bold')
```

Comment:
Should this be a CF string?

Reply:
Fixed in [057e005](https://github.com/island-is/island.is/pull/8301/commits/057e005ab59c28622522f6883c78dba2dfc4ec5b)